README.md

```diff
@@ -9,7 +9,7 @@ Homebrew: `brew tap reverbdotcom/reverb && brew install protopkg`
 With Go 1.11: `go get -u github.com/reverbdotcom/protopkg`
 
 ## Private Repositories
-`protopkg` will make authorized calls to the public github API if the environment variable `GITHUB_TOKEN` is present. Ensure that this token has access to read the given repositories referenced in your `protopkg.json`
+`protopkg` will make authorized calls to the public github API if you set a token via the command line. Ensure that this token has access to read the given repositories referenced in your `protopkg.json`
```

Comment:
Is this another thing devs are going to have to set up?

Reply:
If they update? Yes, they'll have to set their token.

Reply:
I would just use the de facto standard `GITHUB_PULL_TOKEN` that already has instructions on how to set up, is already being verified, etc. Otherwise, if this path simplifies or derisks problems using that, think that

* devs setting up their system should be made aware of this step in the [dev setup instructions](https://github.com/reverbdotcom/docs/blob/master/tech/local-setup.md) so they don't run into a auth problem trying to sync protos in the future 
* devs with existing setups should be able to verify or triage this change with [check-access](https://github.com/reverbdotcom/local/blob/master/Makefile#L89) so they can proactively set this up/find out what's wrong if they miss communication about it/debug issues if there are problems with it